homeassistant/util/async_.py

```diff
@@ -38,7 +40,7 @@ def callback() -> None:
 
 def run_callback_threadsafe(
     loop: AbstractEventLoop, callback: Callable[..., T], *args: Any
-) -> "concurrent.futures.Future[T]":
+) -> concurrent.futures.Future[T]:  # pylint: disable=unsubscriptable-object
```

Comment:
Technically Python 3.9 is required for `concurrent.futures.Future` to be subscriptable. However `mypy` accepts it as it's generic in `typeshed`: [concurrent/futures/_base.pyi](https://github.com/python/typeshed/blob/5b730d4ded6ecf4f72ebe28769e18efb3bdc8996/stdlib/concurrent/futures/_base.pyi#L55-L68)